src/on.ts

```diff
@@ -0,0 +1,97 @@
+import { Handle, EventObject } from './interfaces';
+import { createHandle, createCompositeHandle } from './lang';
+import Evented from './Evented';
+
+// Only used for Evented and EventEmitter, as EventTarget uses EventListener
+type EventCallback = (event: {}) => void;
```

Comment:
Noted. For future reference, how did you discover this? Manually checking the output?